test/integration/narrow/routes/healthz.test.js

```diff
@@ -1,10 +1,10 @@
-import { createServer } from '../../../../app/server.js'
+import { startServer } from '../../../../app/server.js'
 
 describe('healthz test', () => {
   let server
 
   beforeEach(async () => {
-    server = await createServer()
+    server = await startServer()
     await server.initialize()
```

Comment:
same. Be nice to get rid of the before and afters entirely tbh, create the server inside each test?

Reply:
Done, and combined the health endpoints into one test file also